manifests/implementation/runner/terraform/apply-with-state.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
   name: apply-with-state
   license:
     name: "Apache 2.0"
-  displayName: Apply Action for Terraform Runner
-  description: Running Terraform apply with an existing tfstate file
+  displayName: "Apply with existing Terrfarom state"
```

Comment:
In general, YAML doesn't need quotes. More [info] (https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml). Also, I think that the previous version for` displayName` and `description` describes the interface correctly so if  I do not miss something I wouldn't change that.